DQM/SiPixelPhase1Common/src/SiPixelCoordinates.cc

```diff
@@ -0,0 +1,916 @@
+// -*- C++ -*-
```

Comment:
Re memory: I have not measured here, but there are "only" ~1900 Pixel modules in Phase1, so each of the Maps should stay in the range of Kilobytes. There will be one instance of this per Phase1 DQM plugin (and also per thread), so in the end it might be in the lower megabytes.

Re the location: Just to make it clear, this file will not move unless a concrete new path is proposed. I'd happily move it if somebody tells me where to move it...